jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/VirtualThreads.java

```diff
@@ -233,4 +257,7 @@ default void setUseVirtualThreads(boolean useVirtualThreads)
     private VirtualThreads()
     {
     }
+
+    public interface ThreadFactoryExecutor extends ThreadFactory, Executor
```

Comment:
Not a big fan of this... if it can be kept separated, I think it's better.

Reply:
removed, but we are no longer able to implement a Virtual ThreadFactory